default-templates/new-account/profile/card

```diff
@@ -23,3 +28,25 @@
     pim:preferencesFile </settings/prefs.ttl> ;  # private settings/preferences
     solid:publicTypeIndex </settings/publicTypeIndex.ttl> ;
     solid:privateTypeIndex </settings/privateTypeIndex.ttl> .
+
+{{#if addOwlSameAs}}
+:me
+    owl:sameAs <{{extWebId}}> .
+{{/if}}
+
+{{#if tlsCertWebid}}
+:me
+    acl:delegates <{{tlsCertWebid}}> ; # Delegation Relations for Software Agent 
+    cert:key [
+               a cert:RSAPublicKey ;
+               cert:exponent   "65537"^^xsd:int ;
+               cert:modulus    "00"^^xsd:hexBinary
```

Comment:
Just checking did you want to put "00" in here or {{tlsCertModulus}} ?

Reply:
Yes, now `card` profile is create as Delegator profile(that doesn't need PublicKey), so it is something like a stub only for properly work(loading data) with browser YouID extension.

Reply:
That doesn't seem proper usage of modulus then though. Is there any way around this?

Reply:
Yes, it is possible avoid this, I will fix our browser YouID extension